obs-browser-plugin.cpp

```diff
@@ -592,6 +592,12 @@ bool obs_module_load(void)
 	}
 	obs_data_release(private_data);
 #endif
+
+#if defined(__APPLE__)
```

Comment:
I'd suggest

```cpp
#if defined(__APPLE__) && CHROME_VERSION_BUILD < 4183
```

That way this code is discarded when we upgrade CEF.

Ideally, I'd also prefer we had a runtime check for the macOS version so that this code only runs on Big Sur to avoid side effects on older versions.

Reply:
Updated.